src/snowflake/cli/_plugins/nativeapp/artifacts.py

```diff
@@ -728,7 +728,7 @@ def find_and_read_manifest_file(deploy_root: Path) -> Dict[str, Any]:
         "r", read_file_limit_mb=DEFAULT_SIZE_LIMIT_MB
     ) as file:
         manifest_content = safe_load(file.read())
-    return manifest_content
+    return manifest_content or {}
```

Comment:
how can we hit this exactly?

Reply:
looks like safe_load will return None if file is empty